Sources/CoreCommands/Options.swift

```diff
@@ -484,6 +491,32 @@ public struct BuildOptions: ParsableArguments {
         case warn
         case error
     }
+
+    /// An optional intermodule optimization to run at link time.
+    ///
+    /// When using Link Time Optimization (LTO for short) the swift and clang
+    /// compilers produce objects containing containing a higher level
+    /// representation of the program bitcode instead of machine code. The
+    /// linker combines these objects together performing additional
+    /// optimizations with visibility into each module/object, resulting in a
+    /// further optimized version of the executable.
+    ///
+    /// Using LTO can have significant impact on compile times, however can be
+    /// used to dramatically reduce code-size in some cases.
+    ///
+    /// Note: Bitcode objects and machine code objects can be linked together.
+    public enum LinkTimeOptimizationMode: String, Codable, ExpressibleByArgument {
```

Comment:
I think I would prefer referring to the enum in `BuildParameters` instead of duplicating the doc comment.

Reply:
done!